src/Controller/ModulesController.php

```diff
@@ -473,10 +473,55 @@ public function relationshipsJson($id, string $relation) : void
             return;
         }
 
+        $this->getThumbsUrls($response);
+
         $this->set((array)$response);
         $this->set('_serialize', array_keys($response));
     }
 
+    /**
+     * Retrieve thumbnails URL of related objects in `meta.url` if present.
+     *
+     * @param array $response Related objects response.
+     * @return void
+     */
+    protected function getThumbsUrls(array &$response) : void
+    {
+        if (empty($response['data'])) {
+            return;
+        }
+
+        // extract ids of objects
+        $ids = Hash::combine($response, 'data.{n}.id', 'data.{n}.id');
```

Comment:
it seemes you need just the ids... `Hash::extract($response, 'data.{n}.id')` could be fine.